.github/workflows/linters.yml

```diff
@@ -0,0 +1,37 @@
+name: Linters
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  lint-markdown:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run tests
```

Comment:
Only this entry(?) has a name field. The others do not.

Reply:
I removed the solitary name tag as it is optional and not adding much value.